workflow.ipynb

```diff
@@ -138,12 +138,12 @@
   },
```

Comment:
<p>Quarterly is really better because we see longer term trends and it smoothes out the seasonal variability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neovintage/notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>